python/loom_reader.py

```diff
@@ -6,39 +6,45 @@ class LoomReader:
     def __init__(self, filename):
         self.ds = loompy.connect(filename)
 
-        self.cells = list(self.ds.ca['CellID'])
-        self.genes = list(self.ds.ra['Gene'])
+        self.cells = list(self.ds.ca["CellID"])
+        self.genes = list(self.ds.ra["Gene"])
 
     def data(self):
-        '''
+        """
         Returns all data for each cell.
-        '''
+        """
         cells = {}
         data_zip = zip(
-            self.ds.ca['Valid'],
-            self.ds.ca['CellID'],
-            self.ds.ca['_tSNE_1'],
-            self.ds.ca['_tSNE_2'],
-            self.ds.ca['ClusterID'],
-            self.ds.ca['ClusterName'],
-            self.ds.ca['X'],
-            self.ds.ca['Y']
+            self.ds.ca["Valid"],
+            self.ds.ca["CellID"],
+            self.ds.ca["_tSNE_1"],
+            self.ds.ca["_tSNE_2"],
+            self.ds.ca["ClusterID"],
+            self.ds.ca["ClusterName"],
+            self.ds.ca["X"],
+            self.ds.ca["Y"],
         )
-        for (valid, cell_id, tsne1, tsne2,
-             cluster_id, cluster_name, x, y) in data_zip:
+        for (
+            valid,
+            cell_id,
+            tsne1,
+            tsne2,
+            cluster_id,
+            cluster_name,
+            x,
+            y,
+        ) in data_zip:
```

Comment:
Sometimes there is a logical grouping within a larger list: Flattening it all the way out isn't more clear.